adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java

```diff
@@ -1957,6 +1974,9 @@ public void testAutoFlowRefreshTokenRequestFailedWithOauthError() throws IOExcep
                  AuthenticationConstants.UIResponse.BROWSER_CODE_COMPLETE,
                  getResponseIntent(callback, "resource", "clientid", "redirect", TEST_IDTOKEN_UPN));
          signalCallback.await(CONTEXT_REQUEST_TIME_OUT, TimeUnit.MILLISECONDS);
+
+         assertTrue(telemetry.checkOauthError());
```

Comment:
Here you can check the entire set of values that you would be getting after the acquireToken call.Also this can be done for rest of the acquireToken test cases.It will help us get a high test coverage.